regimes/mx/food_vouchers_complement.go

```diff
@@ -0,0 +1,145 @@
+	// List of food vouchers issued to the customer's employees (maps to `Conceptos`).
+	Lines []*FoodVouchersLine `json:"lines" jsonschema:"title=Lines"`
+}
+
+// FoodVouchersLine represents a single food voucher issued to the e-wallet of
+// one of the customer's employees. It maps to one `Concepto` node in the CFDI's
+// complement.
+type FoodVouchersLine struct {
+	// Identifier of the e-wallet that received the food voucher (maps to `Identificador`).
+	EWalletID cbc.Code `json:"e_wallet_id" jsonschema:"title=E-wallet Identifier"`
+	// Date and time of the food voucher's issue (maps to `Fecha`).
+	IssueDateTime cal.DateTime `json:"issue_date_time" jsonschema:"title=Issue Date and Time"`
+	// Employee that received the food voucher.
+	Employee *FoodVouchersEmployee `json:"employee,omitempty" jsonschema:"title=Employee"`
+	// Amount of the food voucher (maps to `importe`).
+	Amount num.Amount `json:"amount" jsonschema:"title=Amount"`
+}
+
+// FoodVouchersEmployee represents an employee that received a food voucher. It
+// groups employee related field that appears under the `Concepto` node in the
+// CFDI's complement.
+type FoodVouchersEmployee struct {
+	// Employee's tax identity code (maps to `rfc`).
+	TaxCode cbc.Code `json:"tax_code" jsonschema:"title=Employee's Tax Identity Code"`
+	// Employee's CURP ("Clave Única de Registro de Población", maps to `curp`).
+	CURP cbc.Code `json:"curp" jsonschema:"title=Employee's CURP"`
+	// Employee's name (maps to `nombre`).
+	Name string `json:"name" jsonschema:"title=Employee's Name"`
+	// Employee's Social Security Number (maps to `numSeguridadSocial`).
+	SocialSecurity cbc.Code `json:"social_security,omitempty" jsonschema:"title=Employee's Social Security Number"`
+}
+
+// Validate checks the FoodVouchersComplement data according to the SAT's
+// rules for the "Complemento de Vales de Despensa".
+func (fvc *FoodVouchersComplement) Validate() error {
+	return validation.ValidateStruct(fvc,
+		validation.Field(&fvc.EmployerRegistration, validation.Length(0, 20)),
+		validation.Field(&fvc.AccountNumber,
+			validation.Required,
+			validation.Length(0, 20),
+		),
+		validation.Field(&fvc.Total, validation.Required),
+		validation.Field(&fvc.Lines,
+			validation.Required,
+			validation.Each(validation.By(validateFoodVouchersLine)),
+		),
+	)
+}
+
+func validateFoodVouchersLine(value interface{}) error {
+	line := value.(*FoodVouchersLine)
+	if line == nil {
+		return nil
+	}
+
+	return validation.ValidateStruct(line,
+		validation.Field(&line.EWalletID,
+			validation.Required,
+			validation.Length(0, 20),
+		),
+		validation.Field(&line.IssueDateTime, cal.DateTimeNotZero()),
+		validation.Field(&line.Employee,
+			validation.Required,
+			validation.By(validateFoodVouchersEmployee)),
+		validation.Field(&line.Amount, validation.Required),
+	)
+}
+
+func validateFoodVouchersEmployee(value interface{}) error {
```

Comment:
BTW, you don't need to manually validate here. You just define the `func (fve *FoodVoucersEmployee) Validate() error` method and ensure the preceding structure includes the field definition:

```
    validation.Field(&line.Employee),
```

Reply:
Nice one! Addressed in cf73168